src/redux/home-page-slides/home-page-slides.operations.js

```diff
@@ -0,0 +1,226 @@
+import { gql } from '@apollo/client';
+import { client } from '../../utils/client';
+import { getFromLocalStorage } from '../../services/local-storage.service';
+import { config } from '../../configs';
+import { slidesTranslations } from '../../translations/home-page-slides.translations';
+
+export const getAllSlides = async (skip, limit) => {
+  const result = await client.query({
+    variables: {
+      skip,
+      limit
+    },
+    query: gql`
+        query($skip: Int, $limit: Int) {
+            getAllSlides(skip: $skip, limit: $limit) {
+                items {
+                    _id
+                    title {
+                        lang
+                        value
+                    }
+                    description {
+                        value
+                        lang
+                    }
+                    images {
+                        large
+                        thumbnail
+                    }
+                    link
+                    show
+                    order
+                }
+                count
+            }
+        }
+    `
+  });
+  // client.resetStore();
```

Comment:
Please remove this useless commented line.

Reply:
Fixed